src/program/processor/new_order.rs

```diff
@@ -371,6 +371,10 @@ fn process_new_order<'a, 'info>(
                     )?;
                 }
             }
+        } else {
+            // Should never execute as the account loading logic should fail
+            phoenix_log!("WARNING: Vault context was not provided");
```

Comment:
Do we have a canonical log format for things that we expect to never get hit? Would be nice so it's easy to set up a log filter and alert.

Reply:
there's only like 70 `phoenix_log!` in the entire repo, we could audit for consistency

happy to punt